docs/node-operators/faq.mdx

```diff
@@ -1,6 +1,13 @@
 ---
 title: Node Operators FAQ
 hide_title: true
+description: Troubleshooting tips and answers to frequently asked questions about node operators in the Mina network.
```

Comment:
This is a very small thing, but does "...frequently asked questions *from* node operators in..." make more sense?

Reply:
both work, in this case, let's use "about" 
(fun fact: I did a web search to see patterns of FAQs and "about" is more common)